container-builder/client/fastmapper.go

```diff
@@ -77,15 +79,15 @@ func newFastDiscoveryRESTMapperWithFilter(config *rest.Config, filter func(*meta
 		wg.Start(func() { discoverGroupResources(dc, gr) })
 	}
 	wg.Wait()
-	logrus.Debugf("Picked %d/%d", pickedCount, totalCount)
+	klog.V(log.D).InfoS("Picked", "count", pickedCount, "totalCount", totalCount)
```

Comment:
Is it klog.V(log.D).InfoS("Picked", "pickedCount", pickedCount, "totalCount", totalCount) ?

Reply:
It can be. I've pushed a commit changing to your proposal.